conda_smithy/azure_ci_utils.py

```diff
@@ -8,6 +8,8 @@
 AZURE_TEAM_INSTANCE = os.getenv("AZURE_INSTANCE", "https://dev.azure.com/conda-forge")
 AZURE_PROJECT_ID = os.getenv("AZURE_PROJECT_ID", "feedstock-builds")
 AZURE_SERVICE_ENDPOINT_NAME = os.getenv("AZURE_SERVICE_ENDPOINT", "conda-forge")
+# Bydefault for now don't report on the build information back to github
```

Comment:
```suggestion
# By default for now don't report on the build information back to github
```

Reply:
@CJ-Wright, do you know how to apply the suggested change?

Reply:
I think @mariusvniekerk needs to push a button somewhere, currently only the PR opener has accept/reject rights